src/NUnitFramework/framework/Api/NUnitTestAssemblyRunner.cs

```diff
@@ -29,9 +29,13 @@
 using NUnit.Framework.Internal.Execution;
 using System.Collections.Generic;
 using System.IO;
+
 #if !NETSTANDARD1_6
 using System.Diagnostics;
 using System.Security;
+#endif
+
+#if NET20 || NET35 || NET40 || NET45
```

Comment:
I probably wrote that WinForms code, but really? 😄

Reply:
We could make this cross-platform... :D